src/Tribe/RSVP.php

```diff
@@ -705,11 +713,18 @@ public function send_tickets_email( $order_id ) {
 		}
 
 		$content     = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ) );
-		$headers     = apply_filters( 'tribe_rsvp_email_headers', array( 'Content-type: text/html' ) );
-		$attachments = apply_filters( 'tribe_rsvp_email_attachments', array() );
-		$to          = apply_filters( 'tribe_rsvp_email_recipient', $to );
+
+		/**
+		 * Filters for RSVP tickets email
```

Comment:
The intent is right, but the way it works in practice is the docblock applies to the line that follows it and when this is parsed by phpDocumenter it will associate this comment with the first filter only ... that to say, the gold standard here would be a docblock per filter _or_ you could alter this one so the description applies to `tribe_rsvp_email_headers` only